packages/backend/src/open_payments/wallet_address/model.ts

```diff
@@ -102,14 +112,17 @@ export class WalletAddress
     authServer: string
     resourceServer: string
   }): OpenPaymentsWalletAddress {
-    return {
+    const returnVal: OpenPaymentsWalletAddress = {
       id: this.url,
       publicName: this.publicName,
       assetCode: this.asset.code,
       assetScale: this.asset.scale,
       authServer,
       resourceServer
     }
+    if (this.additionalProperties)
+      returnVal.additionalProperties = this.additionalProperties
```

Comment:
Could change this to `if (this.additionalProperties.length)` to fix the current CI error.

Although I think another option would be returning `undefined` from the `getAdditionalProperties` service method. Whatever you think makes more sense.